src/opm/parser/eclipse/EclipseState/Schedule/Group/GuideRate.cpp

```diff
@@ -327,4 +322,19 @@ double GuideRate::get_grvalue_result(const GRValState& gr) const
         : std::max(gr.curr.value, 0.0);
 }
 
+
+bool GuideRate::timeToUpdate(const double sim_time, const double time_interval) const
```

Comment:
For good or for bad the `GuideRate` class actually has a `const Schedule&` member - i.e. this method can itself look up the `time_interval` member - no need to pass it in.

Reply:
Sure. it makes sense, will update.